rest/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.profiles.active=@spring.profiles.active@
```

Comment:
Profile hasn't been provided.
What will be values for properties?

Reply:
`spring.profiles.active` value remains `@spring.profiles.active@`
I don't have such profile so It is not used